src/Microsoft.SqlTools.ServiceLayer/Profiler/IProfilerSessionMonitor.cs

```diff
@@ -22,11 +22,16 @@ public interface IProfilerSessionMonitor
         /// <summary>
         /// Starts monitoring a profiler session
         /// </summary>
-        bool StartMonitoringSession(ProfilerSession session);
+        bool StartMonitoringSession(string viewerID, IXEventSession session);
```

Comment:
please name the parameter `viewerId` following .Net framework naming conventions (see here for additional details https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)